comp/networkpath/npcollector/npcollectorimpl/pathteststore/pathteststore.go

```diff
@@ -119,6 +123,11 @@ func (f *Store) Add(pathtestToAdd *common.Pathtest) {
 	f.contextsMutex.Lock()
 	defer f.contextsMutex.Unlock()
 
+	if len(f.contexts) >= f.maxContexts {
```

Comment:
I could see this getting very noisy -- can we count the number of 'exceeded' events we see and log them once in `Flush` ?

Reply:
I changed it so it should only log if it has been a minute since the last time it logged the message. I thought about making it counter based but this feels cleaner